specs/APIReview_AllowDrop.md

```diff
@@ -0,0 +1,98 @@
+# Background
+Currently the drag&drop functionality is default enabled in webview2 and there is no way to disable it. Some developers may want to disbale this functionality in their applications 
+based upon their scenarios. According to such feature requirements, we add the new API to provide developers with the capability to configure the drag&drop functionality.
+
+# Description
+We add a new `AllowDrop` property in `CoreWebView2Controller`. This API allows end developers to toggle the drag&drop functionality easily.
+If it's disabled, any drag&drop actions will keep out of work. By default, it's disabled to accommodate to conventions.
+
+# Examples
+## C++
+
+```cpp
+void ToggleAllowDrop()
+{
+    // Get webView's controller
+    wil::com_ptr<ICoreWebView2Controller> controller = m_appWindow->GetWebViewController();
+    if (controller)
+    {
+        BOOL allowDrop;
+        CHECK_FAILURE(controller->get_AllowDrop(&allowDrop));
+        if (allowDrop)
+        {
+            CHECK_FAILURE(controller->put_AllowDrop(FALSE));
+            MessageBox(
+                nullptr, L"WebView disallows dropping files now.",
+                L"WebView AllowDrop property changed", MB_OK);
+        }
+        else
+        {
+            CHECK_FAILURE(controller->put_AllowDrop(TRUE));
+            MessageBox(
+                nullptr, L"WebView allows dropping files now.",
+                L"WebView AllowDrop property changed", MB_OK);
+        }
+    }
+}
+```
+
+## C#
+```c#
+void ToggleAllowDrop(object target, ExecutedRoutedEventArgs e)
+{
+    // Get webView's controller
+    var controller = _webView.CoreWebView2Controller;
+    if(controller.AllowDrop)
```

Comment:
Small issue: missing space between if and (
```suggestion
    if (controller.AllowDrop)
```

Reply:
fixed